tests/common.go

```diff
@@ -3740,10 +3747,11 @@ func CreateAzureBucket(bucketName string) {
 }
 
 // createAzureBackupLocation creates backup location
-func createAzureBackupLocation(name, uid, cloudCred, cloudCredUID, bucketName, orgID string) {
-	Step(fmt.Sprintf("Create Azure backup location [%s] in org [%s]", name, orgID), func() {
-		CreateAzureBackupLocation(name, uid, cloudCred, cloudCredUID, bucketName, orgID)
-	})
+func createAzureBackupLocation(name, uid, cloudCred, cloudCredUID, bucketName, orgID string) error {
+	log.InfoD("Create Azure backup location [%s] in org [%s]", name, orgID)
+	err := CreateAzureBackupLocation(name, uid, cloudCred, cloudCredUID, bucketName, orgID)
+	return err
+
```

Comment:
you can just return, since you not doing anything else with the `err` you getting
```
CreateAzureBackupLocation(name, uid, cloudCred, cloudCredUID, bucketName, orgID)
```

Reply:
done